com.esrille.furiganapad.json

```diff
@@ -0,0 +1,30 @@
+{
+    "id" : "com.esrille.furiganapad",
```

Comment:
The website is at https://www.esrille.com/ (just a public note)

Reply:
this will be troublesome for the verification process afaik.

Reply:
We have enabled the redirect from our naked domain to ‘www.’ It will take effect within 48 hours.